app/helpers/pickup_libraries_helper.rb

```diff
@@ -20,35 +22,70 @@ def label_for_pickup_libraries_dropdown(pickup_libraries)
 
   private
 
-  def select_for_multiple_libraries(form, pickup_libraries)
-    return unless pickup_libraries.length > 1
+  # Symphony
+  def select_for_multiple_destinations(form, pickup_destinations)
+    return unless pickup_destinations.length > 1
 
     form.select(
       :destination,
-      pickup_libraries_array(pickup_libraries),
+      pickup_destinations_array(pickup_destinations),
       {
-        label: label_for_pickup_libraries_dropdown(pickup_libraries),
-        selected: form.object.destination || form.object.default_pickup_library
+        label: label_for_pickup_destinations_dropdown(pickup_destinations),
+        selected: form.object.destination || form.object.default_pickup_destination
       },
       aria: { controls: 'scheduler-text' },
       data: { 'paging-schedule-updater' => 'true', 'text-selector' => "[data-text-object='#{form.object.object_id}']" }
     )
   end
 
-  def single_library_markup(form, library)
+  def single_destination_markup(form, pickup_destination)
+    destination_label = get_destination_label(pickup_destination)
     <<-HTML
       <div class='form-group'>
         <div class='#{label_column_class} control-label'>
-          #{label_for_pickup_libraries_dropdown([])}
+          #{label_for_pickup_destinations_dropdown([])}
         </div>
         <div class='#{content_column_class} input-like-text'>
-          #{Settings.libraries[library]&.label || library}
+          #{destination_label || pickup_destination}
         </div>
-        #{form.hidden_field :destination, value: library}
+        #{form.hidden_field :destination, value: pickup_destination}
       </div>
     HTML
   end
 
+  # Get the label, if it exists, for the pickup destination
+  def get_destination_label(pickup_destination)
```

Comment:
```suggestion
  def destination_label(pickup_destination)
```
I think this is a bit more conventional. See https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/AccessorMethodName